contracts/RiskFund/ProtocolShareReserve.sol

```diff
@@ -36,13 +44,53 @@ contract ProtocolShareReserve is OwnableUpgradeable, ExponentialNoError {
         riskFund = _riskFund;
     }
 
+    /**
+     * @dev Update the reserve of the asset for the specific pool after transferring to protocol share reserve.
+     * @param comptroller  Comptroller address(pool).
+     * @param asset Asset address.
+     */
+    function updateAssetsState(address comptroller, address asset) external {
+        require(
+            comptroller != address(0),
+            "Liquidated shares Reserves: Comptroller address invalid"
+        );
+        require(
+            asset != address(0),
+            "Liquidated shares Reserves: Asset address invalid"
+        );
+        uint256 currentBalance = IERC20(asset).balanceOf(address(this));
+        if(currentBalance > previousStateForAssets[asset]) {
```

Comment:
Do we need to handle the case where the currentBalance is less than the previous state?

Reply:
No, currentBalance cannot be less than previous state.